src/main/java/org/openrewrite/kotlin/internal/KotlinPrinter.java

```diff
@@ -432,6 +432,19 @@ public J visitUnary(K.Unary unary, PrintOutputCapture<P> p) {
         return unary;
     }
 
+    @Override
+    public J visitUseSite(K.UseSite useSite, PrintOutputCapture<P> p) {
+        beforeSyntax(useSite, Space.Location.ANNOTATIONS, p);
+        visit(useSite.getTarget(), p);
+        visitSpace(useSite.getPadding().getTarget().getAfter(), Space.Location.ANNOTATIONS, p);
```

Comment:
I would have expected some kind of SUFFIX space location here. Probably specific to K.UseSite.

Reply:
I think the `JRightPadded<NameTree> target` is enough here, as the right padding is the space before the colon.
```kotlin
@field /*Right padding of the target*/ : Ann1
```